db/migrations/6_responder_queue.sql

```diff
@@ -0,0 +1,27 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE responder_queue (
+    id UUID DEFAULT generate_ulid() PRIMARY KEY,
+    responder_id UUID NOT NULL,
+    error TEXT NULL,
+    created_at timestamp NOT NULL DEFAULT now(),
+    updated_at timestamp NOT NULL DEFAULT now()
+);
+
+CREATE OR REPLACE FUNCTION insert_responder_queue()
+RETURNS TRIGGER AS $$
+BEGIN
+    INSERT INTO responder_queue(responder_id) VALUES (new.id);
+    SELECT pg_notify("responder_updates", new.id);
```

Comment:
I don't think we can use pg_notify like this - It could cause updates to be duplicated, rather pg_notify should notify the go process to run a `select * from responder_queue with SKIP locked` e.g. https://www.enterprisedb.com/blog/what-skip-locked-postgresql-95 

That will prevent duplicate delivery and ensure all messages are delivered

Reply:
Okay, interesting. We don't have to change anything here though, the query would be changed on the implementation end.